File: app/src/main/java/com/example/ftapp11/FinancialTrackerApplication.java
package com.example.ftapp11;

import android.app.Application
        import com.example.ftapp11.AppContainer
                import com.example.ftapp11.
public class FinancialTrackerApplication : Application(){
    lateinit var container: AppContainer

        override fun onCreate() {
        super.onCreate()
        container = AppDataContainer(this)
        }
}

File: app/src/main/java/com/example/ftapp11/FinancialTrackerApplication.kt
package com.example.ftapp11

import android.app.Application
import com.example.ftapp11.data.AppContainer
import com.example.ftapp11.data.AppDataContainer

class FinancialTrackerApplication : Application() {
    lateinit var container: AppContainer

    override fun onCreate() {
        super.onCreate()
        container = AppDataContainer(this)
    }
}